.github/workflows/CrossRefCheckWithAnchors.yml

```diff
@@ -0,0 +1,12 @@
+name: CrossRefCheck with Anchors
+
+on: 
+  push:
+    branches:
+      - review
+
+jobs:
+  crossref-check-with-anchors:
+    uses: Open-Systems-Pharmacology/Workflows/.github/workflows/MarkdownCrossrefCheck_withAnchors.yml@main
+    with:
+      folder-path: 'Evaluation/report'
```

Comment:
Adding the standard checks.
(When creating a new model repository from the OSP Model Repository template those are added automatically)